exercises/Python_Principles_3.ipynb

```diff
@@ -3104,192 +2716,7 @@
    ]
   },
   {
-   "cell_type": "markdown",
```

Comment:
@AlexTeslenko It looks like these questions are being removed on merge of this branch to main. This is what I think happened in Chris did his latest commit. I'll add them back in for you to quickly review.

Reply:
thanks for noticing this @medic-code !